src/validations/rules/sap.sch

```diff
@@ -559,6 +568,60 @@
                 role="error"
                 test="oscal:party[oscal:prop[@ns = 'https://fedramp.gov/ns/oscal' and @name = 'iso-iec-17020-identifier' and matches(@value, '^\d{4}\.\d{2}$')]]">This
                 FedRAMP SAP has a metadata/party with an @name of 'iso-iec-17020-identifier' has a correctly formatted @value.</sch:assert>
+
+            <sch:assert
+                diagnostics="has-metadata-csp-poc-role-diagnostic"
+                doc:guide-reference="Guide to OSCAL-based FedRAMP Security Assessment Plans (SAP) §4.12"
+                fedramp:specific="true"
+                id="has-metadata-csp-poc-role"
+                role="error"
+                test="oscal:role[@id = 'csp-assessment-poc']">This FedRAMP SAP has a role with an @id value of 'csp-assessment-poc'.</sch:assert>
+
+            <sch:assert
+                diagnostics="has-metadata-csp-poc-responsible-party-diagnostic"
+                doc:guide-reference="Guide to OSCAL-based FedRAMP Security Assessment Plans (SAP) §4.12"
+                fedramp:specific="true"
+                id="has-metadata-csp-poc-responsible-party"
+                role="error"
+                test="oscal:responsible-party[@role-id = 'csp-assessment-poc']">This FedRAMP SAP has a responsible-party with an @role-id value of
+                'csp-assessment-poc'.</sch:assert>
+
+            <sch:assert
+                diagnostics="has-metadata-responsible-party-csp-poc-matches-diagnostic"
+                doc:guide-reference="Guide to OSCAL-based FedRAMP Security Assessment Plans (SAP) §4.12"
+                fedramp:specific="true"
+                id="has-metadata-responsible-party-csp-poc-matches"
+                role="error"
+                test="
+                    if (not(oscal:responsible-party[@role-id = 'csp-assessment-poc']/oscal:party-uuid = $SSP-partyPersonIDs))
+                    then
+                        if (oscal:responsible-party[@role-id = 'csp-assessment-poc']/oscal:party-uuid = $partyPersonIDs)
+                        then
+                            true()
+                        else
+                            false()
+                    else
+                        true()"
```

Comment:
The context for this should be `responsible-party[@role-id = 'csp-assessment-poc']`, not `metadata`.

The test can be reduced (with that context) to `oscal:party-uuid = ($SSP-partyPersonIDs, $SAP-partyPersonIDs)`.

Reply:
Yes.  For clarity this assertion is moved to a new rule.